src/containers/HSM/HSMListV2/HSMListV2.module.css

```
@@ -1,185 +1,227 @@
-  gap: 8px;
-  flex-shrink: 0;
+  flex-direction: column;
+  row-gap: 0.4rem;
 }
 
-.SyncWrapper {
+.LabelText {
+  font-weight: 600;
+  font-size: 16px;
+  color: #0c1f14;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)